src/utils/graphql/queries/getAllLookupTableStructures.query.ts

```diff
@@ -2,11 +2,11 @@ import { gql } from '@apollo/client'
 
 export default gql`
   query getAllLookupTableStructures {
-    lookupTables {
+    dataTables(condition: { isLookupTable: true }) {
```

Comment:
This is how we distinguish "lookup tables" from other custom data tables. (since we don't want other data tables to be editable, at least not here)